### DISCUSSION TITLE
0.14 early access query

### DISCUSSION THREAD

**Discussion by leccelecce** [blakeblackshear/frigate#9420] [General]
I have a spare USB coral/server. Interested in trying out the 0.14 branch, though it seems packages aren't being published from it.

If I was feeling super keen and happy to try, is there any way to get an image other than pulling the whole branch via git and building it locally?

Appreciate probably not looking for issues to be raised until there is an actual 0.14-alpha or beta

**Comment by NickM-27** [comment]
Yes, there are many ui bugs and nothing is close to decided on. You just build it yourself if you want to try it.

**Reply by leccelecce** [reply]
Thanks, turned out to be simpler than I thought to build a local Docker image: `git clone` followed by `make` on my Debian Bookworm system.
